content/post/mnist-example.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "A Scikit-learn pipeline in Wallaroo"
+draft = false
+date = 2017-11-16T00:00:00Z
+tags = [
+    "wallaroo",
+    "machine learning",
+    "python",
+    "example",
+    "tutorial"
+]
+categories = [
+    "Machine Learning"
+]
+description = "Creating an inference pipeline with MNIST"
+author = "amosca"
++++
+
+Nowadays, many applications with streaming data are either applying machine learning or have a very good use case for it. In this example, we will explore how we can build a sample pipeline to classify images from the [MNIST dataset](ihttp://yann.lecun.com/exdb/mnist/), using PCA and a Logistic Regression in scikit-learn. As a prerequisite, you should be familiar with some notions of machine learning, and some notions about Wallaroo pipelines.
```

Comment:
Why are `many applications with streaming data are either applying machine learning or have a very good use case for it`? This needs more context for the uninformed reader.

It would be great if you could expand on "some notions of machine learning". As someone who knows a very tiny bit about ML, I have no clue if I know enough or not for this example (unless "some notions" means "PCA and logistic regression" from the sentence prior, in which case I didn't make that connection).

Additionally, the "notions about Wallaroo pipelines" should link to either an earlier blog post or the documentation at a minimum. It would be great if we can add in a line or two summary of what Wallaroo is/allows. Maybe something like "wallaroo is our streaming platform that allows developers to write in either Python or Go and autoscale easily."

Reply:
It would also be great to have a one or two line summary of `scikit-learn` along with a link to it for folks who want to read more.